projector_type: "TransverseMercator"
map_origin:
  latitude: 35.61739731
  longitude: 139.7797546
